.github/workflows/docker.yml

```diff
@@ -258,7 +122,7 @@ jobs:
         cache-from: type=local,src=/tmp/.buildx-cache
         cache-to: type=local,dest=/tmp/.buildx-cache
     - name: Slack Notification
-      if: ${{ steps.prep.outputs.slack-notification-version && steps.prep_base_version.outputs.should-notify-slack == 'true' && job.status == 'success' }}
+      if: ${{ steps.prep.outputs.slack-notification-version && steps.prep_base_version.outputs.disable-slack-notification == 'false' && job.status == 'success' }}
```

Comment:
Yes, but it's the only one.

I decided to change it because I noticed during the extraction that `should-notify-slack` could be misleading sometimes, because we could set it to `true` just to set it to `false` afterwards during the execution.

With `disable-slack-notification` this doesn't happen and the code becomes clearer.

But I could perform this change in the base branch as well, if needed.

Reply:
I guess we can keep it here. This change is missing from the Acceptance Criteria. Would you please add it there?

Another thing, change the commit message to "chore(ci): Change `should-notify-slack` to `disable-slack-notification` and extract scripts from the workflow definition".

Reply:
Done